### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by xckitahara**
This PR Update README.
- Added about the build method.
- Deleting the description of subcommands deleted in #14.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
  limitations under the License.
 
 
+# Build
 
+## Prerequisites
+
+- git
```
Comment (federkasten) [inline README.md:32]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
  limitations under the License.
 
 
+# Build
```
Comment (federkasten) [inline README.md]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
  limitations under the License.
 
 
+# Build
 
+## Prerequisites
```
Comment (federkasten) [inline README.md]:
```suggestion
# Prerequisites

```

File: README.md [hunk 4]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
  limitations under the License.
 
 
+# Build
 
+## Prerequisites
+
+- git
+- JDK >= 1.8 (Recommended 11)
+
```
Comment (federkasten) [inline README.md:34]:
```suggestion

# Build
```

File: README.md [hunk 5]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
...
+- JDK >= 1.8 (Recommended 11)
+
+```
+$ git clone https://github.com/genome-rcast/karkinos.git
+$ cd karkinos
+$ ./gradlew uberjar
+```
+
+Gradle does not need to be pre-installed.
+
```
Comment (federkasten) [inline README.md]:
```suggestion
You don't need to install Gradle command.
```

**Review by federkasten — COMMENTED** [review]
Thanks @xckitahara.

I suggested some trivial points and the sentence structure.
Additionally, please fix the following.

- L101 and L102 are duplicated descriptions for the `analysis` subcommand.
- L103 describes deleted `reanalysis` subcomamnd.

File: README.md [hunk 6]
```diff
@@ -90,8 +105,6 @@ command
 
 For single computational node use analysis command to excute all genomic range.
 
-If parallel computational nodes are avilavle, use analysis command to each chromosome and use reanalysis command to maerge the result.
-
 Example shell script is on the web sites.
```
Comment (federkasten) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (alumi) [reply README.md:32]: We need git not only for cloning the repository but for executing `./gradlew uberjar`

File: README.md [hunk 7]
```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
...
+$ git clone https://github.com/genome-rcast/karkinos.git
+$ cd karkinos
+$ ./gradlew uberjar
+```
+
+Gradle does not need to be pre-installed.
+
+`karkinos-standalone-X.Y.Z-SNAPSHOT.jar` is created in the `./build/libs/` directory.
 
 # Install
```
Comment (federkasten) [inline README.md]: I think this "Install" section is outdated and no longer necessary.

**Review by alumi — CHANGES_REQUESTED** [review]
We don't have those commands anymore
https://github.com/genome-rcast/karkinos/pull/22/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R103-R104
Remove them and update the surrounding texts so that it looks natural with a single subcommand

File: README.md (hunk 1 above)
Comment (federkasten) [reply README.md:32]: Sorry. I don't know Gradle uses Git internally :open_mouth:

**Comment by xckitahara** [comment]
Thank you for the review.
I fixed.

**Review by federkasten — DISMISSED** [review]
:ok_man:

File: README.md [hunk 8]
```diff
@@ -82,21 +92,15 @@ example line
 
 # Run
 
-command
-
-- analysis - pileup and analysis SNV,CNV,Tumor purity
-- analysis - pileup only if chr is disgnated
-- reanalysis - analysis SNV,CNV,Tumor purity from piled up files.
+pileup and analysis SNV,CNV,Tumor purity
```
Comment (alumi) [inline README.md]:
```suggestion
The current version of karkinos supports only one subcommand, `analysis`.
This subcommand will pileup reads then analyze SNVs, CNVs, and Tumor purity.
```

File: README.md [hunk 9]
```diff
@@ -82,21 +92,15 @@ example line
...
-- analysis - pileup only if chr is disgnated
-- reanalysis - analysis SNV,CNV,Tumor purity from piled up files.
+pileup and analysis SNV,CNV,Tumor purity
 
-For single computational node use analysis command to excute all genomic range.
-
-If parallel computational nodes are avilavle, use analysis command to each chromosome and use reanalysis command to maerge the result.
+```
+$ java -jar karkinos.jar analysis
+```
```
Comment (alumi) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -82,21 +92,15 @@ example line
...
 
-Example shell script is on the web sites.
+options
```
Comment (alumi) [inline README.md]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -82,21 +92,15 @@ example line
...
 
 ```
-karkinos.jar analysis
-
 usage: karkinos.jar analysis -n <arg> -t <arg> -r <arg> -snp <arg> -ct
```
Comment (alumi) [inline README.md]:
```suggestion
usage: java -jar karkinos.jar analysis -n <arg> -t <arg> -r <arg> -snp <arg> -ct
```

**Review by alumi — CHANGES_REQUESTED** [review]

**Review by alumi — APPROVED** [review]

**Pull request merged**